### ISSUE TITLE
Problem with pre-trained model

### ISSUE THREAD

**Issue by mentian** [zju3dv/clean-pvnet#82]
HI, Sida,
I had a problem with loading the pre-trained model with pytorch 0.4.1. Do you have any suggestion on how to resolve it?
Detailed info:
I could successfully load the pre-trained model, and reproduce the test results with both pvnet (another repo, pytorch 0.4.1) and clean-pvnet (pytorch 1.1.0). But I failed to load the model to clean-pvnet with pytorch 0.4.1.  With exactly same input, the outputs of intermediate layers are different from those which lead to correct pose prediction. I assume there is no randomness in your code, therefore I guess the problem should be "model loading". Somehow, the loaded model are different between pytorch 1.1.0 and 0.4.1.

**Comment by mentian** [comment]
Problem resolved.
pvnet does not set model.eval() during inference, while clean-pvnet does so. That's why the intermediate outputs are different. Both can give correct pose estimation (and they should). However, setting trained model to evaluation mode is recommended.
The real problem of clean-pvnet when using pyotrch 0.4.1 is [here](https://github.com/zju3dv/clean-pvnet/blob/master/lib/csrc/ransac_voting/ransac_voting_gpu.py). After I replace it with the corresponding code of pvnet, everything works fine. I guess it's because of "b_Inv", but not very sure. Further efforts are needed to locate the problem.
